compass/sign/RemoteSignatureSource.java

```diff
@@ -81,10 +108,17 @@ private static SslContext buildSslContext(
   @Override
   public String getSignature(long index, String hash) {
     log.trace("Requesting signature for index: " + index + " and hash: " + hash);
-
-    GetSignatureResponse response = serviceStub.getSignature(GetSignatureRequest.newBuilder().setIndex(index).setHash(hash).build());
-
-
+    GetSignatureResponse response;
+    try {
+      response = serviceStub.getSignature(GetSignatureRequest.newBuilder().setIndex(index).setHash(hash).build());
+    } catch (StatusRuntimeException e) {
+      // If an exception occurs, wait 10 seconds, and retry only once by rebuilding the gRPC client stub from a new Channel
+      try {
+        Thread.sleep(10_000);
+      } catch (InterruptedException ex) {}
```

Comment:
Can you use `finally` here instead of catch?
Or at least write a comment inside the block that you swallow the exception?

I just find it more readable.